<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBookPro" failures="1" tests="6" name="tests.loginFunctionality.LoginTest" time="71.732" errors="0" timestamp="2025-02-24T00:43:55 GMT+04:00" skipped="0">
  <testcase classname="tests.loginFunctionality.LoginTest" name="testWrongCredentials" time="8.987"/>
  <testcase classname="tests.loginFunctionality.LoginTest" name="LoginWithEmptyFields" time="14.091"/>
  <testcase classname="tests.loginFunctionality.LoginTest" name="testInvalidEmailFormat" time="6.591">
    <failure message="Error message was not displayed for empty fields." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Error message was not displayed for empty fields.
at org.testng.Assert.fail(Assert.java:111)
at tests.loginFunctionality.LoginTest.testInvalidEmailFormat(LoginTest.java:157)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
    </failure>
  </testcase> <!-- testInvalidEmailFormat -->
  <testcase classname="tests.loginFunctionality.LoginTest" name="testLogout" time="20.601"/>
  <testcase classname="tests.loginFunctionality.LoginTest" name="testInsensivity" time="14.191"/>
  <testcase classname="tests.loginFunctionality.LoginTest" name="testCorrectCredentials" time="7.271"/>
</testsuite> <!-- tests.loginFunctionality.LoginTest -->
